package moscow.rockstar.framework.objects.gradient.impl;

import moscow.rockstar.framework.objects.gradient.Gradient;
import moscow.rockstar.util.colors.ColorRGBA;

public class HorizontalGradient extends Gradient {
   public HorizontalGradient(ColorRGBA startColor, ColorRGBA endColor) {
      super(startColor, startColor, endColor, endColor);
   }

   public HorizontalGradient rotate() {
      return new HorizontalGradient(this.bottomRightColor, this.topLeftColor);
   }
}
